src/lambda_function.py

```diff
@@ -1,61 +1,82 @@
-import os
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this
+# software and associated documentation files (the "Software"), to deal in the Software
+# without restriction, including without limitation the rights to use, copy, modify,
+# merge, publish, distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+# INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+# PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+# HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+# OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+# SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+#
+
 import json
 import logging
+import os
 
 
 # Set LogLevel using environment variable, fallback to INFO if not present
 logger = logging.getLogger()
-try:
-    log_level = os.environ['LogLevel']
-    if log_level not in ['INFO', 'DEBUG']:
-        log_level = 'INFO'
-except:
+log_level = os.getenv('LogLevel', 'INFO')
```

Comment:
Why was the try/except removed?

Reply:
not needed with 'getenv' -- which defaults to 'INFO' if the key isn't found.  Accomplishes the same result with more economy